google/api/http.proto

```diff
@@ -346,6 +347,20 @@ message HttpRule {
   // message type.
   string body = 7;
 
+  // A mapping from HTTP headers to request fields. If
```

Comment:
A mapping from HTTP request headers to gRPC request fields. The map key corresponds to an HTTP header name (case-insensitive), and the map value corresponds to an gRPC request field path (using the same syntax as the path template).